transfer-eth-erc20/to-near.js

```diff
@@ -330,6 +349,15 @@ class TransferETHERC20ToNear {
     )
 
     if (transferLog.finished === undefined) {
+      // TODO fix before using
+      // Mint tokens first???
+      /*await ethERC20Contract.methods
+        .mint(ethSenderAccount, Number(amount))
+        .send({ from: ethSenderAccount, gas: 5000000 })*/
```

Comment:
I think we don't mint in eth side, and not all erc20 token has mint, instead we use approve+lock